development/src/pages/home/page.home.njk

```diff
@@ -27,27 +27,27 @@
 {% endblock header %}
 
 {% block main %}
-{% call sectioncomponent('Professional Skillset', 'skillset') %}
-     <ul class="skill-List">
+{% call sectioncomponent('Professional Skillset', 'skills') %}
+     <ul class="skills__listgroup">
```

Comment:
@Adititomar123 please fix the indentation

Reply:
done